### ISSUE TITLE
Alerts: Can't Change Theme Color of Close (X) Icons

### ISSUE THREAD

**Issue by ahand** [vmware/clarity#3016]
## Describe the bug

Currently, the Alert's Close (X) icon is a gray color and its active color appears to be white. 

In the Alerts section of the Dark Theme file, there doesn't seem to be a way to change the color of the Close (X) icon on the regular alerts. There is a property for the App Level Alert's Close icon called:  $clr-app-alert-close-icon-color. 

For the regular Alerts, there is a property called  $clr-alert-action-color, but this affects right side action links and drop downs, not the Close icon. 

Since there is a property for the App level alert's Close icon, I would expect to see a similar property for the regular alert icon. For example, something like: $clr-alert-close-icon-color. 

## How to reproduce

I made changes to the properties shown in the file:  
clarity-master/src/clr-angular/utils/_theme.dark.clarity.scss

Examples of values: 
$clr-theme-alert-font-color: #000000;

// Info type
$clr-alert-info-bg-color: #F5FBFE;
$clr-alert-info-font-color:  $clr-theme-alert-font-color;  //ok. local variable
$clr-alert-info-border-color:  #0070BA;  
$clr-alert-info-icon-color: #0070BA;  

$clr-alert-action-color: #000000;  // Used for right side action links & dropdowns
$clr-alert-action-active-color: #000000;// Alert dropdowns when they are clicked on
$clr-app-alert-close-icon-color: #ffffff; //Only used on App Level Alerts, which makes sense. 

1. In the Alerts section of the Dark Theme file, set up values for the regular Alert's Info type, as well as the other relevant alert properties.
2.  Run the Clarity Kitchen Sink app.
3.  View the Static Alerts section of the demo. 
4.  You'll see that the Info Alert's Close icon is gray, and that there doesn't seem to be a way to set it among the other properties surfaced here. 

## Expected behavior

Consider surfacing a property for the regular Alert's Close icons, as is done for the App Level alerts. Or maybe I've overlooked the relevant property?  

## Versions

Kitchen Sink app as downloaded from Git on Monday, Jan. 7, 2019.

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: Appears to be v1.0.3

**Device:**

* MacOS, Chrome

![alert - close button color issue](https://user-images.githubusercontent.com/11655671/51003983-a637f300-14ed-11e9-81f0-04d9d379c4ed.png)

**Comment by ahand** [comment]
OK. I did some investigation. It appears that the Close icon's normal and hover effect colors for the standard Alerts come from the global properties for such. When I wrote this ticket, I didn't fully understand what the global properties were used for, and expected that the Close icon colors would also be separately available. It would be useful to label either the sample Dark Theme with this info, and maybe add a note in the documentation somewhere. 

These two properties affect the standard Alert's Close icon in normal and hover states. 

$close-color--normal: #DE0063; //pink
$close-color--hover: #00CF92; //green

This issue may be considered for closure.
